### ISSUE TITLE
Execute/access remote workflows on KNIME Server

### ISSUE THREAD

**Issue by applio** [knime/knimepy#11]
It would be useful to be able to mirror the existing functionality for working with local KNIME Workflows to be able to work with remote Workflows on a KNIME Server.

**Comment by applio** [comment]
The same motivations for issue #24 should also be addressed when working with remote workflows.

**Comment by applio** [comment]
Workflows with no "Container Table (Output)" node may execute successfully but attempted processing of the output results in an exception (thanks Franziska for reporting this and the issue described in my prior comment):
```
File ".../site-packages/knime.py", line 598, in execute
    for output in rest_service_output["outputValues"].values():
KeyError: 'outputValues'
```

**Comment by applio** [comment]
Previously mentioned issue #24 is now also addressed for remote workflows in commit 7ad5d7b403dd804698022c3785b5717d337e5a8e.  Thanks goes to Franziska for pointing this out.

**Comment by applio** [comment]
Workflows with no "Container Table (Output)" node may now be processed properly after successful execution as of commit 23f16311fbe6f1275965be422ed11c0da9b78305.  Thanks again to Franziska for reporting this.
